packages/experimental/Button/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@fluentui-react-native/experimental-button",
+  "version": "0.1.0",
+  "description": "A cross-platform Button component using the Fluent Design System",
+  "main": "src/index.ts",
+  "module": "src/index.ts",
+  "typings": "lib/index.d.ts",
+  "private": true,
+  "onPublish": {
+    "main": "lib-commonjs/index.js",
+    "module": "lib/index.js"
+  },
+  "scripts": {
+    "build": "fluentui-scripts build",
+    "just": "fluentui-scripts",
+    "clean": "fluentui-scripts clean",
+    "update-snapshots": "fluentui-scripts jest -u",
+    "verify-api": "fluentui-scripts verify-api-extractor",
+    "update-api": "fluentui-scripts update-api-extractor"
+  },
+  "dependencies": {
+    "@uifabricshared/foundation-compose": "^1.6.70",
```

Comment:
will this dep change to the experimental version?

Reply:
Yes, that will be part of the later change.